src/python/pants/option/options.py

```diff
@@ -5,16 +5,55 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 import copy
+import re
 import sys
 from builtins import object, open, str
 
-from pants.base.deprecated import warn_or_error
+from pants.base.deprecated import get_frame_info, warn_or_error
 from pants.option.arg_splitter import GLOBAL_SCOPE, ArgSplitter
 from pants.option.global_options import GlobalOptionsRegistrar
 from pants.option.option_util import is_list_option
 from pants.option.option_value_container import OptionValueContainer
 from pants.option.parser_hierarchy import ParserHierarchy, all_enclosing_scopes, enclosing_scope
 from pants.option.scope import ScopeInfo
+from pants.util.memo import memoized_property
+from pants.util.objects import datatype
+
+
+class CallableWrapper(datatype(['callable_object'])):
```

Comment:
I don't think that this class is necessary... a "callable wrapper" is easiest expressed with just an anonymous function?

Reply:
I don't think so either, I was able to remove it in ~10 seconds so it obviously wasn't doing anything special. See fb95e63972ee356bc3d1a7c23b361a88f2dae720!